src/sentry/services/hybrid_cloud/project/impl.py

```diff
@@ -30,6 +31,15 @@ def get_by_id(self, *, organization_id: int, id: int) -> RpcProject | None:
             return serialize_project(project)
         return None
 
+    def get_many_by_organizations(
+        self, *, region_name: str, organization_ids: List[int]
+    ) -> List[RpcProject]:
+        projects = Project.objects.filter(
+            organization__in=organization_ids,
+            status=ObjectStatus.ACTIVE,
+        )
```

Comment:
Getting an unbounded result set from multiple orgs could go sideways down the road. Should we limit this to the most recent 50 or 100 projects?